released/packages/coq-dpdgraph/coq-dpdgraph.0.6.6/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "yves.bertot@inria.fr"
+license: "LGPL 2.1"
+
+homepage: "https://github.com/karmaki/coq-dpdgraph"
+
+build: [
+  ["./configure"]
+  ["echo" "%{jobs}%" "jobs for the linter"]
+  [make "WARN_ERR="]
+ ]
+
+bug-reports: "https://github.com/karmaki/coq-dpdgraph/issues"
+
+dev-repo: "git+https://github.com/karmaki/coq-dpdgraph.git"
+install: [
+  [make "install" "BINDIR=%{bin}%"]
+]
+
+remove: [
```

Comment:
As per above, the whole `remove` should be deleted.

Reply:
@palmskog thanks for the review and suggestions.  I was a bit frustrated by the github interface that after using the UI to accept your suggestions as commits, I was unable to receives this modifications in my local working tree.  I re-did the modification, added the suggestion about removing the useless 'remove' field, and force-pushed.  I hope it does not break too many things.